sky/skylet/constants.py

```diff
@@ -130,6 +130,7 @@
 # backend_utils.write_cluster_config.
 RAY_SKYPILOT_INSTALLATION_COMMANDS = (
     'mkdir -p ~/sky_workdir && mkdir -p ~/.sky/sky_app;'
+    'export PIP_DISABLE_PIP_VERSION_CHECK=1;'
```

Comment:
Why do we need this here?

Reply:
Added a comment. This is to avoid the distracting pip python check warning message.